qpmodel/LogicNode.cs

```diff
@@ -686,56 +686,7 @@ public override List<int> ResolveColumnOrdinal(in List<Expr> reqOutput, bool rem
             var rtables = rchild_().InclusiveTableRefs();
             var lreq = new HashSet<Expr>();
             var rreq = new HashSet<Expr>();
-            // allocate the output to related child respectively
-            Debug.Assert((this.lchild_() != null) && (this.rchild_()!=null));
-            if(this is LogicMarkJoin)
-            {
-                var lchild = this.lchild_();
-                var rchild = this.rchild_();
-                List<Expr> reqFromChildRemove = new List<Expr>();
-                lchild.VisitEach(x =>
-                {
-                    if (x is LogicMarkJoin)
-                    {
-                        foreach (var v in reqFromChild)
-                        {
-                            if (v is MarkerExpr)
-                            {
```

Comment:
This condition means that there exists( #marker & #marker ), it means we need to assign the #marker to whitch child it belongs to. 
However, currentlly handling method will  filter the #marker when it show up. 
So there will be no several #markers.
We will miss some chance to optimize the markerjoin orders.
So there should be some TODO items.

Reply:
But, the utimate plan is confirmed after the joinoder optimization. 
and when a #marker is produced, there can be a filter to filter the #marker .  
So it seems this branch handling #marker1 & #maker2 will be no use.